build.sbt

```diff
@@ -287,3 +287,8 @@ lazy val bench = project
   .dependsOn(spark)
   .settings(commonSettings)
   .settings(Settings.bench)
+
+lazy val `geotrellis-tiling` = project
```

Comment:
We have no subprojects called `geotrellis-X` since all subprojects are `geotrellis` subprojects. Mb just to name it tiling?

Reply:
:+1: